config/core/deployments/activator-hpa.yaml

```diff
@@ -45,8 +48,12 @@ metadata:
   namespace: knative-serving
   labels:
     serving.knative.dev/release: devel
+    app.kubernetes.io/name: activator
+    app.kubernetes.io/version: devel
+    app.kubernetes.io/part-of: knative
 spec:
   minAvailable: 80%
   selector:
     matchLabels:
-      app: activator
+      app.kubernetes.io/name: activator
+      app.kubernetes.io/version: devel
```

Comment:
I don't think we should change selectors in this release - I think that'll break upgrading

Reply:
🤔 , `spec.selector` is immutable

Reply:
I have reverted the deployment/service selectors